### PR TITLE
Fix verification when user includes a space in their email.

### PR DESCRIPTION
**Pull request by DFurnes**
#### Changes
Some users were being _SLOPPY_ and including a trailing space in their emails. The nerve! This normalizes those values before validating or checking for them in the database. :relieved: 

---
For review: @angaither @weerd

### CODE REVIEW

File: app/Auth/Registrar.php [hunk 1]
```diff
@@ -60,7 +60,7 @@ public function login($credentials)
     public function normalize($credentials)
     {
         if (! empty($credentials['email'])) {
-            $credentials['email'] = strtolower($credentials['email']);
+            $credentials['email'] = trim(strtolower($credentials['email']));
```
Comment (angaither) [inline app/Auth/Registrar.php]: do we want this in mobile too?

**Comment by angaither** [comment]
:+1:

**Comment by weerd** [comment]
:+1:

File: app/Auth/Registrar.php (hunk 1 above)
Comment (DFurnes) [inline app/Auth/Registrar.php]: I think the regex should handle it, but should probs make a test for that too!

**Pull request merged**